packages/hop-node/src/watchers/classes/ContractBase.ts

```diff
@@ -1,10 +1,17 @@
 import rateLimitRetry from 'src/decorators/rateLimitRetry'
-import { BigNumber, Contract, Transaction, providers } from 'ethers'
+import { BigNumber, Contract, Transaction as EthersTransaction, providers } from 'ethers'
 import { Chain } from 'src/constants'
 import { EventEmitter } from 'events'
 import { chainIdToSlug, chainSlugToId, wait } from 'src/utils'
 import { config } from 'src/config'
 
+export class NotFoundError extends Error {}
```

Comment:
What is the purpose of this new Error class?

Reply:
it's an artifact from something i was trying earlier. Should be removed now